packages/expo-secure-store/plugin/src/withSecureStore.ts

```diff
@@ -1,19 +1,58 @@
-import { ConfigPlugin, IOSConfig, createRunOncePlugin } from '@expo/config-plugins';
+import {
+  ConfigPlugin,
+  IOSConfig,
+  createRunOncePlugin,
+  withAndroidManifest,
+  AndroidConfig,
+} from '@expo/config-plugins';
```

Comment:
```suggestion
} from 'expo/config-plugins';
```

let's revise the import by using re-exported config-plugins from the `expo` package.